universal-application-tool-0.0.1/app/services/program/predicate/Operator.java

```diff
@@ -2,25 +2,31 @@
 
 /** Represents a JsonPath operator (https://github.com/json-path/JsonPath#filter-operators). */
 public enum Operator {
-  ANY_OF("anyof"),
-  EQUAL_TO("=="),
-  GREATER_THAN(">"),
-  GREATER_THAN_OR_EQUAL_TO(">="),
-  IN("in"),
-  LESS_THAN("<"),
-  LESS_THAN_OR_EQUAL_TO("<="),
-  NONE_OF("noneof"),
-  NOT_EQUAL_TO("!="),
-  NOT_IN("nin"),
-  SUBSET_OF("subsetof");
+  ANY_OF("anyof", "is any of"),
+  EQUAL_TO("==", "is equal to"),
+  GREATER_THAN(">", "is greater than"),
+  GREATER_THAN_OR_EQUAL_TO(">=", "is greater than or equal to"),
+  IN("in", "is in"),
```

Comment:
suggestion: "is one of" - this is the case where there is a single value on the left and an array on the right

Reply:
Oops - I missed this on my first pass. Done!

Reply:
Thanks for your attention to detail on these display strings.